### ISSUE TITLE
setup fails when LC_ALL=C is set

### ISSUE THREAD

**Issue by canufeel** [peopledoc/django-chartjs#19]
I believe the issue here is that when LC_ALL=C is set the encoding is switched to `encodings/ascii.py` and we get an error which is `UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 880: ordinal not in range(128)`. Here is the error trace:

```
File "/var/www/venv/eis/build/django-chartjs/setup.py", line 19, in <module>
long_description=read_relative_file('README.rst'),
File "/var/www/venv/eis/build/django-chartjs/setup.py", line 11, in read_relative_file
return f.read()
File "/var/www/venv/eis/lib/python3.4/encodings/ascii.py", line 26, in decode
return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 880: ordinal not in range(128)
```

Because ascii does not support some symbols used in the html which is the part of this README.rst we get the error. This is not an issue in the most of cases but makes impossible to install the module via some provisioning systems like SaltStack.

**Comment by Alwerdani** [comment]
+1

**Comment by Natim** [comment]
Oh yes, sorry about that I know how to fix it. Thanks for noticing.

**Comment by Natim** [comment]
Should be fixed with a6bdfe4
